phpunit.dusk.xml

```diff
@@ -6,7 +6,7 @@
          convertErrorsToExceptions="true"
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
-         processIsolation="true"
```

Comment:
@nekodex, do you remember why this is set to `true`? Checking [the dusk template](https://github.com/laravel/dusk/blame/6.x/stubs/phpunit.xml) looks like it's always `false`. Is it some leftover from experiments?

Reply:
I have a feeling it was to try and work-around something weird but I don't remember exactly what, sorry.

If setting it to false doesn't cause any issues now, it's probably fine to change?

Reply:
I'm actually getting some weirdness where the test sometimes runs as the wrong user causing the expectations to fail but I haven't been able to isolate it yet 😕